evm_body.yaml

```diff
@@ -173,9 +173,7 @@ eth_getTransactionCount:
     - type: object
       properties:
         address:
-          type: string
-          description: 20 Bytes - Address
-          pattern: '^0[xX][0-9a-fA-F]{40}$'
+          $ref: '#/AddressParam'
         blockNumberOrTagOrHash:
           $ref: '#/BlockNumberOrTagOrHash'
```

Comment:
the properties / inputs here have to be wrapped into a `params` array

Reply:
Fixed the issue, but had to go back to the former approach because different types were not getting displayed as elements of the same array. 

I've tested all the APIs after fixing the issue, and all the APIs that are expected to work are now working. 

You can also test all of them here: 

- [eth_getBalance](https://alchemy-api-docs.readme.io/reference/eth-getbalance-2)
- [eth_getStorageAt](https://alchemy-api-docs.readme.io/reference/eth-getstorageat-1)
- [eth_getTransactionCount](https://alchemy-api-docs.readme.io/reference/eth-gettransactioncount-1)
- [eth_getCode](https://alchemy-api-docs.readme.io/reference/eth-getcode-1)
- [eth_call](https://alchemy-api-docs.readme.io/reference/eth-call-2)
- [eth_getProof](https://alchemy-api-docs.readme.io/reference/eth-getproof-2)

Be sure to change the API key while testing because `docs-demo` won't work from this domain.